### PR TITLE
Try to make the review job uniq

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Now each review job overwrites the previous status. Let's make it uniq.

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 683a731fd39f5b3c93194a380c2a141e22432cac with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/56987/683a731fd39f5b3c93194a380c2a141e22432cac/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/56987/683a731fd39f5b3c93194a380c2a141e22432cac/ci_running.html">✅ success</a></td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6927383495">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Review by Felixoid — COMMENTED** [review]
Let's test

File: .github/workflows/pull_request_approved.yml [hunk 1]
```diff
@@ -21,3 +22,7 @@ jobs:
         run: |
           cd "$GITHUB_WORKSPACE/tests/ci"
           python3 merge_pr.py --check-approved
+          cat << 'EOF'
```
Comment (Felixoid) [inline .github/workflows/pull_request_approved.yml:25]: another test

Comment (Felixoid) [reply .github/workflows/pull_request_approved.yml:25]: and another one

Comment (Felixoid) [reply .github/workflows/pull_request_approved.yml]: and final

Comment (Felixoid) [reply .github/workflows/pull_request_approved.yml]: and just to check

**Comment by Felixoid** [comment]
It didn't work out

**Pull request closed without merging**